src/components/layout/Group/Group.jsx

```diff
@@ -1,14 +1,25 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+// @flow
+/* eslint react/prop-types: 0 */
+import React, { type Node } from 'react';
 import cx from 'classnames';
 import { map, isArray, toArray } from 'lodash';
-import {
-  getDataAttrs,
-  getDataProps,
-} from '../../../dataUtils';
+import { getDataAttrs } from '../../../dataUtils';
+import type { Data } from '../../../types';
 import style from './style.scss';
 
-class Group extends React.Component {
+type Props = {|
+    children: Array<Node>,
```

Comment:
just `Node`

Reply:
Have to use Array<Node> as this.props.children.length is being accessed. Flow complains on using just Node.